movie-app/src/body/Searched.js

```diff
@@ -4,14 +4,14 @@ const Searched = ({ movies }) => {
     const moviesCheck = () => {
         if (movies){
             return movies.map(movie => {
-                return <MovieCard key={movie.imdbID} movie={movie}/> 
+                return <MovieCard key={movie.imdbID} movie={movie}/>
```

Comment:
SMART!  If you key it by the `imdbID`, you get the added bonus of their database's primary key uniqueness invariant.